### PR TITLE
Adding avsc.dino.icu

### PR DESCRIPTION
**Pull request by AVSCGREAT**
# [Adding/Deleting/Updating] `subdomain.[hackclub.com/dino.icu/etc]`

## Description

adding avsc.dino.icu, for hosting my personal website soon. currently the weather app i created is hosted here. pls approve.

### CODE REVIEW

File: hackclub.com.yaml [hunk 1]
```diff
@@ -319,7 +319,7 @@ adastra:
 astra:
   - ttl: 600
     type: CNAME
-    value: astra-collective.github.io.
+    value: astra2-ops.github.io.
```
Comment (alx-alexpark) [inline hackclub.com.yaml:322]: What is this change?

Comment (astra2-ops) [reply hackclub.com.yaml:322]: image
Hi the chang in the the above img was as i shifted the the github user 
image
this above shown change is me (from @AVSCGREAT ) adding my to be personal website

**Comment by AVSCGREAT** [comment]
Correct @astra2-ops

**Comment by astra2-ops** [comment]
***any update***

File: hackclub.com.yaml (hunk 1 above)
Comment (alx-alexpark) [reply hackclub.com.yaml:322]: I'm still confused on what this subdomain is for? Is it an official HQ project or a club?

Comment (alx-alexpark) [reply hackclub.com.yaml:322]: @AVSCGREAT @astra2-ops ^

**Review by alx-alexpark — APPROVED** [review]
looks ok

**Review by alx-alexpark — CHANGES_REQUESTED** [review]
Actually could you resolve the merge conflict? Didn't see it sorry

**Pull request merged**